<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherSettings</class>
 <widget class="QDialog" name="LauncherSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launcher Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>551</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Close|QDialogButtonBox::StandardButton::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="LauncherSetDescGroupBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>431</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>Setting Description</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTextBrowser" name="LauncherSetDescText">
      <property name="autoFormatting">
       <set>QTextEdit::AutoFormattingFlag::AutoNone</set>
      </property>
      <property name="lineWrapColumnOrWidth">
       <number>0</number>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;BBLauncher Update:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Updates to the newest release of BBLauncher if there is one newer than the current version. Checking updates at startup seems to slow down startup of BBLauncher a little.&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Check Portable Settings:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Checks if portable settings folders (&amp;quot;user&amp;quot; folders) exist in the shadPS4 and BBLauncher folders, prompt to create symlinks if none exist (not available for no-UAC version)&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Themes:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Changes the color scheme of the launcher.&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;60 FPS Sound Fix:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Fixes some sounds not working after a crash when the 60 FPS patch is enabled.&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Backup Saves:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Automatically transfers Bloodborne saves to backup folders in the BBLauncher/Backups folder. BACKUP1 will always be the newest, then BACKUP2 and so on. You can restore backups folders from the Save Manager.&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Create Direct Shortcut:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Creates a shortcut to directly launch Bloodborne with BBLauncher's settings and mods, skipping the GUI&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextInteractionFlag::NoTextInteraction</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>541</width>
     <height>412</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BBLauncher Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>BBLauncher</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Updates</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QCheckBox" name="UpdateCheckBox">
                <property name="text">
                 <string>Check BBLauncher updates at startup</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="UpdateButton">
                <property name="text">
                 <string>Check BBLauncher updates</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="PortableSettingsCheckBox">
             <property name="text">
              <string>Check Portable Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="shortcutButton">
             <property name="text">
              <string>Create Direct Shortcut</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ThemeGroupBox">
          <property name="title">
           <string>Themes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="DarkThemeRadioButton">
             <property name="text">
              <string>Dark Theme</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="LightThemeRadioButton">
             <property name="text">
              <string>Light Theme</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Hacks</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="SoundFixCheckBox">
             <property name="text">
              <string>60 FPS Sound Fix</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>shadPS4 Updates</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>shadPS4 Update Channel</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QComboBox" name="shadChannelComboBox">
                <item>
                 <property name="text">
                  <string>Nightly</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Release</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="shadUpdateCheckBox">
             <property name="text">
              <string>Check shadPS4 updates at startup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="shadUpdateButton">
             <property name="text">
              <string>Check shadPS4 updates</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="DownloadProgressBar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="BackupSaveGroupBox">
          <property name="title">
           <string>Backup Saves</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="BackupSaveCheckBox">
             <property name="text">
              <string>Generate backup saves</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="BackupIntervalLabel">
             <property name="text">
              <string>Interval (minutes)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="BackupIntervalComboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="BackupNumberLabel">
             <property name="text">
              <string>Number of backups</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="BackupNumberComboBox"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LauncherSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LauncherSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
